R/odbc-connection.R

```diff
@@ -140,11 +140,15 @@ NULL
 
 #' odbcConnectionColumns
 #'
+#' @description
+#' `r lifecycle::badge("deprecated")`
```

Comment:
I'd suggest you recommend `dbListFields()` here too.

Reply:
You got it! 9abd87a.